pages/CreatingAnExploration.html

```diff
@@ -9,20 +9,22 @@
 
   <p>
     After you've played an exploration, why not try your hand at making one? To
-    do so, navigate to the gallery, and look for the "Create Exploration"
+    do so, navigate to the gallery, and look for the "Create"
     button in the top navbar.
   </p>
 
   <img src="static/images/galleryCreateButton.png" style="width: 100%;">
 
   <p>
-    This will open a dialog box.
+    This will take you to the exploration editor page, click on the Setting icon on the top right 
+    corner of the page; this will take you to a page where in you fill the basic settings 
+    for your exploration.
   </p>
 
   <img src="static/images/createNewExploration.png" style="width: 100%;">
```

Comment:
By default does this show the red error messages? If not, use a screenshot that does not show them. (You want the experience to be similar to that of someone who follows the instructions in this page exactly.)